pandas/tests/tseries/offsets/test_offsets.py

```diff
@@ -678,6 +678,10 @@ def test_isAnchored_deprecated(self, offset_types):
         expected = off.is_anchored()
         assert result == expected
 
+    def test_offsets_hashable(self, offset_types):
+        # GH: 37267
+        assert hash(offset_types) is not None
```

Comment:
This needs an additional `off = self._get_offset(offset_types)`, because now you are testing that the class is hashable, not the instance.

Reply:
Thanks, done